### ISSUE TITLE
ContextMenuTrigger overrides onMouseDown and other attributes

### ISSUE THREAD

**Issue by julienw** [vkbansal/react-contextmenu#105]
In https://github.com/vkbansal/react-contextmenu/blob/769ad7fe83c3745e229111550228a11682c2e3c3/src/ContextMenuTrigger.js#L98-L107 the attributes `onContextMenu`, `onMouseDown`, `onMouseUp`, `onTouchStart`, `onTouchEnd`, `onMouseOut` are overridden. As a result we can't set any of them to trigger separate behaviors, even if we don't need them (for example with `holdToDisplay` set to a negative value).

A good fix would be to call them if present in `attributes`. For example with onMouseDown:
```javascript
    handleMouseDown = (event) => {
        ... // current implementation

        callIfExists(this.props.attributes.onMouseDown, event);
  }
```

**Comment by vkbansal** [comment]
Can you open a PR, with the suggested changes?

**Comment by julienw** [comment]
I opened https://github.com/vkbansal/react-contextmenu/pull/107

**Comment by julienw** [comment]
When do you think you'll be able to trigger a new version ? Thanks !

**Comment by vkbansal** [comment]
Done. Released v2.5.2
